tests/Application/config/sylius/1.9/packages/_sylius.yaml

```diff
@@ -0,0 +1,39 @@
+imports:
+    - { resource: "@SyliusCoreBundle/Resources/config/app/config.yml" }
```

Comment:
In skeleton, you have only this line and other lines are redundant in my opinion.